tests/Suggest/PhraseTest.php

```diff
@@ -46,6 +46,29 @@ public function testToArray(): void
      * @group functional
      */
     public function testPhraseSuggest(): void
+    {
+        $suggest = new Suggest();
+        $phraseSuggest = new Phrase('suggest1', 'text');
+        $phraseSuggest->setText('elasticsearch is bansai coor');
+        $phraseSuggest->setAnalyzer('simple')->setHighlight('<suggest>', '</suggest>')->setStupidBackoffSmoothing(0.4);
+        $phraseSuggest->addDirectGenerator(new DirectGenerator('text'));
+        $suggest->addSuggestion($phraseSuggest);
+
+        $index = $this->_getIndexForTest();
+        $result = $index->search($suggest);
+        $suggests = $result->getSuggests();
+
+        // 3 suggestions should be returned: One in which both misspellings are corrected, and two in which only one misspelling is corrected.
+        $this->assertCount(3, $suggests['suggest1'][0]['options']);
+
+        $this->assertEquals('elasticsearch is <suggest>bonsai cool</suggest>', $suggests['suggest1'][0]['options'][0]['highlighted']);
+        $this->assertEquals('elasticsearch is bonsai cool', $suggests['suggest1'][0]['options'][0]['text']);
+    }
+
+    /**
+     * @group functional
+     */
```

Comment:
```suggestion
     * @group legacy
     */
```

Reply:
And probably add a check for deprecation triggering (using the proper trait): 
```
$this->expectDeprecation('Since ruflin/elastica 7.2.0: XXXX . It will be removed in 8.0.');
```